_posts/2016-07-07-Rust-1.10.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "Announcing Rust 1.10"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.10. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.10][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.10][notes] on GitHub.
+1276 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1100-2016-07-07
+
+### What's in 1.10 stable
+
+Rust 1.10 contains one of the most-requested features in Rust: through the [`-C
+panic=abort` flag] or a [setting in your `Cargo.toml`], when a `panic!` happens,
+it will abort rather than unwind. Why would you want to do this? With an
+abort, less code gets generated, meaning that binary sizes are a bit smaller,
+and compilation time is ever-so-slightly faster. You give up the ability to
```

Comment:
Can we be a bit more quantitative here? I bet @alexcrichton has some representative numbers.